app/src/main/java/com/chesire/nekome/ActivityViewModel.kt

```diff
@@ -20,10 +24,24 @@ import kotlin.coroutines.CoroutineContext
 class ActivityViewModel @Inject constructor(
     private val repo: AccessTokenRepository,
     private val logoutHandler: LogoutHandler,
+    private val settings: ApplicationSettings,
     @IOContext private val ioContext: CoroutineContext,
     userRepository: UserRepository
 ) : ViewModel() {
 
+    private val navigationChannel = Channel<NavDirections>(Channel.BUFFERED)
+    val navigation = navigationChannel.receiveAsFlow()
+    private val snackBarChannel = Channel<Unit>()
+    val snackBar = snackBarChannel.receiveAsFlow()
+
+    init {
```

Comment:
Would it be possible to add a couple of tests for this new `init` block?

Reply:
Wrote one test, but did not work as expected.

I have created a mock observer and used `veryfy` to see of `onChanged` was called. THe problem is that the test fails, saying `Verification failed: call 1 of 1: Observer(#8).onChanged(eq(ActionOnlyNavDirections(actionId=2131230963)))) was not called`

Any hints to fix this? 

why do you use ` every { onChanged(any()) } just Runs` for every observer? On my project we have lots of flaky tests and I blame those observers. Did you also has such issues?